packages/core/useFileDialog/index.ts

```diff
@@ -0,0 +1,53 @@
+import { readonly, ref } from 'vue-demi'
+
+export interface UseFileDialogOptions {
+  multiple?: boolean
+  accept?: string
+}
```

Comment:
```suggestion
export interface UseFileDialogOptions {
  multiple?: boolean
  accept?: string
  capture?: string
}
```

new options `capture`